gcc/config/riscv/riscv.md

```diff
@@ -1813,8 +1813,8 @@
 })
 
 (define_insn "*movhf_hardfloat"
-  [(set (match_operand:HF 0 "nonimmediate_nonpostinc" "=f,   f,f,f,m,m,*f,*r,  *r,*r,*m")
-	(match_operand:HF 1 "move_operand"         " f,zfli,G,m,f,G,*r,*f,*G*r,*m,*r"))]
+  [(set (match_operand:HF 0 "nonimmediate_nonpostinc" "=f,   f,f, f,am,m,*f,*r,  *r,*r,*m")
+	(match_operand:HF 1 "move_operand"            " f,zfli,G,am, f,G,*r,*f,*G*r,*m,*r"))]
```

Comment:
Unrelated whitespace change here and a few times below.

Reply:
This was so the formatting matched upstream. Should I revert it?